packages/app/src/app/overmind/namespaces/checkout/constants.ts

```diff
@@ -1,50 +1,123 @@
-import { CreditAddon, SandboxAddon } from './state';
-
-export const CREDIT_ADDONS: CreditAddon[] = [
-  {
-    id: 'credits_500',
-    type: 'credits',
-    credits: 500,
-    price: 9,
-  },
-  {
-    id: 'credits_4000',
-    type: 'credits',
-    credits: 4000,
-    price: 50,
-    fullPrice: 72,
-    discount: 30,
-  },
-  {
-    id: 'credits_24000',
-    type: 'credits',
-    credits: 24000,
-    price: 216,
-    fullPrice: 432,
-    discount: 50,
-  },
-];
-export const SANDBOX_ADDONS: SandboxAddon[] = [
-  {
-    id: 'sandboxes_100',
-    type: 'sandboxes',
-    sandboxes: 100,
-    price: 9,
-  },
-  {
-    id: 'sandboxes_400',
-    type: 'sandboxes',
-    sandboxes: 400,
-    price: 25,
-    fullPrice: 36,
-    discount: 30,
-  },
-  {
-    id: 'sandboxes_1500',
-    type: 'sandboxes',
-    sandboxes: 1500,
-    price: 68,
-    fullPrice: 136,
-    discount: 50,
-  },
-];
+import { PricingPlan, CreditAddon, SandboxAddon } from './types';
+
+// All values below are the base hardcoded values
+// They get updated with the results from /api/pricing
+
+export const FREE_PLAN: PricingPlan = {
+  id: 'free',
+  name: 'Free',
+  price: 0,
+  credits: 400,
+  creditsNote: 'Ideal for hobbyists using Devboxes up to 40 hours a month.',
+  sandboxes: 20,
+  storage: 20,
+  usage: [
+    'Up to 40 hours worth of VM credits per month',
+    '20 GB storage per VM',
+    'VMs up to 4 vCPUs + 8 GB RAM',
+    'Unlimited Devboxes and repositories',
+    '20 Sandboxes',
+    '10 personal drafts',
+    '5 members',
+  ],
+  features: [
+    'Private Sandboxes, Devboxes & repos',
+    'Codeium AI code-completion',
+    'Live collaboration',
+    'VS Code extension',
+    'Instant environment resume',
+    'Instant environment share',
+  ],
+};
+
+export const PRO_PLAN: PricingPlan = {
+  id: 'flex',
+  name: 'Pro',
+  price: 9,
+  priceDiscountNote: 'Early access discount',
+  credits: 1000,
+  creditsNote:
+    'Ideal to get started with cloud development and understand your usage needs.',
+  sandboxes: 100,
+  storage: 50,
+  features: ['All free features'],
+  usage: [
+    'Start from 100 hours worth of VM credits per month',
+    '50 GB storage per VM',
+    'VMs up to 16 vCPUs + 32 GB RAM',
+    'Unlimited Devboxes and repositories',
+    '100 Sandboxes',
+    'Unlimited personal drafts',
+    '20 members',
+    'Access on-demand VM credits for $0.18 per hour',
+  ],
+};
+
+export const ENTERPRISE_PLAN: PricingPlan = {
+  id: 'enterprise',
+  name: 'Enterprise',
+  price: 0,
+  credits: 0,
+  sandboxes: 0,
+  storage: 0,
+  usage: [],
+  features: [
+    'Unlimited members',
+    'Unlimited API',
+    'VMs up to 64 vCPUs + 128 GB RAM',
+    'On-premise options',
+    'Private managed cloud',
+    'Dedicated support',
+    'SSO',
+  ],
+};
+
+export const ADDON_CREDITS_500: CreditAddon = {
```

Comment:
Can't we get this from endpoint too?

Reply:
good catch! I made another commit with extracting the addon values, I forgot about them at the end

Reply:
So maybe we can remove them from `constants.ts` to avoid future confusion

Reply:
ah no, they are still needed there as the constants define the initial value and we store some additional things on the client, like the percentage deduction or the price before the deduction

Reply:
actually, my bad, we override credits and price, which is all we care about and type is kinda redundant there (I think). I'd still keep the values here because it doesn't hurt to have something on page load